src/core/Akka.Remote.Tests/Serialization/DaemonMsgCreateSerializerSpec.cs

```diff
@@ -72,7 +72,7 @@ public void Serialization_must_serialize_and_deserialize_DaemonMsgCreate_with_De
                 ConfigurationFactory.ParseString("a=1"),
                 new RoundRobinPool(5, null, supervisorStrategy, null),
                 new RemoteScope(new Address("akka", "Test", "host1", 1921)),
-                "mydispatcher");
+                "mydispatcher").WithStashCapacity(10);
```

Comment:
To reproduce whether or not `StashCapacity` was made serializable correctly for remote deployments - this test failed until I updated the `DaemonMsgCreateSerializer`.